ckanext/canada/tables/hospitalityq.yaml

```diff
@@ -608,15 +613,19 @@ resources:
           errors := errors || ARRAY[['start_date', {start_end_date_error}]];
         END IF;
 
-        IF NEW.employee_attendees IS NOT NULL AND NEW.employee_attendees::int < 0 THEN
+        IF NEW.end_date >= '2025-04-01'::date THEN
+          errors := errors || required_error(NEW.disclosure_group, 'disclosure_group');
+        END IF;
+
+        IF NEW.end_date >= '2025-04-01'::date AND NEW.employee_attendees IS NOT NULL AND NEW.employee_attendees::int < 0 THEN
```

Comment:
employee_attendees is already an int so I don't believe the cast does anything here